docker/entrypoint.sh

```diff
@@ -1,128 +1,292 @@
+        # Run a Zebra full sync test on testnet.
+        run_cargo_test "${ENTRYPOINT_FEATURES}" "full_sync_testnet"
+        # List directory generated by test
+        # TODO: replace with ${ZEBRA_CACHED_STATE_DIR} in Rust and workflows
+        list_directory "/zebrad-cache"
+
+      elif [[ "${TEST_DISK_REBUILD}" -eq "1" ]]; then
+        # Run a Zebra sync up to the mandatory checkpoint.
+        #
+        # TODO: use environmental variables instead of Rust features (part of #2995)
+        run_cargo_test "test_sync_to_mandatory_checkpoint_${NETWORK,,},${ENTRYPOINT_FEATURES}" "sync_to_mandatory_checkpoint_${NETWORK,,}"
+        # TODO: replace with ${ZEBRA_CACHED_STATE_DIR} in Rust and workflows
+        list_directory "/zebrad-cache"
+
+      elif [[ "${TEST_UPDATE_SYNC}" -eq "1" ]]; then
+        # Run a Zebra sync starting at the cached tip, and syncing to the latest tip.
+        #
+        # List directory used by test
+        list_directory "${ZEBRA_CACHED_STATE_DIR}"
+        run_cargo_test "${ENTRYPOINT_FEATURES}" "zebrad_update_sync"
+
+      elif [[ "${TEST_CHECKPOINT_SYNC}" -eq "1" ]]; then
+        # Run a Zebra sync starting at the cached mandatory checkpoint, and syncing past it.
+        #
+        # List directory used by test
+        # TODO: replace with ${ZEBRA_CACHED_STATE_DIR} in Rust and workflows
+        list_directory "/zebrad-cache"
+        # TODO: use environmental variables instead of Rust features (part of #2995)
+        run_cargo_test "test_sync_to_mandatory_checkpoint_${NETWORK,,},${ENTRYPOINT_FEATURES}" "sync_past_mandatory_checkpoint_${NETWORK,,}"
+
+      elif [[ "${GENERATE_CHECKPOINTS_MAINNET}" -eq "1" ]]; then
+        # Generate checkpoints after syncing Zebra from a cached state on mainnet.
+        #
+        # TODO: disable or filter out logs like:
+        # test generate_checkpoints_mainnet has been running for over 60 seconds
+        #
+        # List directory used by test
+        list_directory "${ZEBRA_CACHED_STATE_DIR}"
+        run_cargo_test "${ENTRYPOINT_FEATURES}" "generate_checkpoints_mainnet"
+
+      elif [[ "${GENERATE_CHECKPOINTS_TESTNET}" -eq "1" ]]; then
+        # Generate checkpoints after syncing Zebra on testnet.
+        #
+        # This test might fail if testnet is unstable.
+        #
+        # List directory used by test
+        list_directory "${ZEBRA_CACHED_STATE_DIR}"
+        run_cargo_test "${ENTRYPOINT_FEATURES}" "generate_checkpoints_testnet"
+
+      elif [[ "${TEST_LWD_RPC_CALL}" -eq "1" ]]; then
+        # Starting at a cached Zebra tip, test a JSON-RPC call to Zebra.
+        list_directory "${ZEBRA_CACHED_STATE_DIR}"
+        # Run both the fully synced RPC test and the subtree snapshot test, one test at a time.
+        # Since these tests use the same cached state, a state problem in the first test can fail the second test.
+        run_cargo_test "${ENTRYPOINT_FEATURES}" "fully_synced_rpc_"
```

Comment:
This line is missing the test-threads argument. It's ok to add that argument to run_cargo_test, because all the other tests only run one test function. (So the number of threads doesn't matter.)

Reply:
Thank you! Fixed with [fix: add missing test-threads to fully_synced_rpc_](https://github.com/ZcashFoundation/zebra/pull/7660/commits/0f0c175ded0f1e79a20a9f9dcb5f193b9aa838c5)